dipy/data/fetcher.py

```diff
@@ -591,6 +597,125 @@ def read_syn_data():
 """
 
 
+def fetch_viz_icons():
+    """ Download icons for visualization
+    """
+    url = 'https://dl.dropboxusercontent.com/u/2481924/'
+    fname = 'icomoon.tar.gz'
+    icomoon = url + fname
+    folder = pjoin(dipy_home, 'icons')
+
+    url_list = [icomoon]
+    md5_list = ['94a07cba06b4136b6687396426f1e380']
+    fname_list = [fname]
+
+    if not os.path.exists(folder):
+        print('Creating new directory %s' % folder)
+        os.makedirs(folder)
+        print('Downloading icons ...')
+        for i in range(len(md5_list)):
+            _get_file_data(pjoin(folder, fname_list[i]), url_list[i])
+            new_path = pjoin(folder, fname_list[i])
+            check_md5(new_path, md5_list[i])
+            ar = tarfile.open(new_path)
+            ar.extractall(path=folder)
+            ar.close()
+
+        print('Done.')
+        print('Files copied in folder %s' % folder)
+    else:
+        msg = 'Dataset is already in place. If you want to fetch it again, '
+        msg += 'please first remove the folder %s '
+        print(msg % folder)
+
+
+def read_viz_icons(style='icomoon', fname='infinity.png'):
+    """ Read specific icon from specific style
+
+    Parameters
+    ----------
+    style: str
+        Current icon style. Default is icomoon.
+    fname: str
+        Filename of icon. This should be found in folder HOME/.dipy/style/.
+        Default is infinity.png.
+
+    Returns
+    --------
+    path: str
+        Complete path of icon.
+
+    """
+
+    folder = pjoin(dipy_home, 'icons', style)
+    return pjoin(folder, fname)
+
+
+def fetch_bundles_2_subjects():
+    """ Download 2 subjects with their bundles
+    """
+    url = 'https://dl.dropboxusercontent.com/u/2481924/'
+    fname = 'bundles_2_subjects.tar.gz'
+    url = url + fname
+    folder = pjoin(dipy_home, 'exp_bundles_and_maps')
+
+    url_list = [url]
+    md5_list = ['97756fbef11ce2df31f1bedf1fc7aac7']
+    fname_list = [fname]
+
+    if not os.path.exists(folder):
+        print('Creating new directory %s' % folder)
+        os.makedirs(folder)
+        print('Downloading dataset ...')
```

Comment:
We usually tell people how much data they're downloading. Could we do that here as well? I am running the example and  it seems like it's a lot of data coming down, but I don't know exactly how much...